servers/quarkus-server/pom.xml

```diff
@@ -200,6 +200,43 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.projectnessie</groupId>
+      <artifactId>nessie-versioned-persist-mongodb</artifactId>
+      <version>${project.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongodb-crypt</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.github.jnr</groupId>
+      <artifactId>jnr-unixsocket</artifactId>
+      <version>0.38.11</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.luben</groupId>
+      <artifactId>zstd-jni</artifactId>
+      <version>1.4.8-1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
+      <version>1.1.8.4</version>
+    </dependency>
```

Comment:
This is required for the native image build to be able to resolve java linkages. 

IDK why Quarkus cannot infer that from Maven manifests.

Reply:
Have you tried to use the `quarkus-mongodb-client` module?

BTW: Versions should all be defined in the top-level pom

Reply:
Yes, that helped - thanks! 8a1ee77c980aad7108562e2ce8294f30b9dd7557